[2021-04-25 00:08:48,532] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: bash_operator_dag.second_task 2021-04-06T00:00:00+00:00 [queued]>
[2021-04-25 00:08:48,565] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: bash_operator_dag.second_task 2021-04-06T00:00:00+00:00 [queued]>
[2021-04-25 00:08:48,566] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-04-25 00:08:48,566] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2021-04-25 00:08:48,566] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-04-25 00:08:48,648] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): second_task> on 2021-04-06T00:00:00+00:00
[2021-04-25 00:08:48,667] {standard_task_runner.py:52} INFO - Started process 1856 to run task
[2021-04-25 00:08:48,685] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'bash_operator_dag', 'second_task', '2021-04-06T00:00:00+00:00', '--job-id', '654', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/simple_bash_operator_dag.py', '--cfg-path', '/tmp/tmplcaiherx', '--error-file', '/tmp/tmp6rsrsbgc']
[2021-04-25 00:08:48,686] {standard_task_runner.py:77} INFO - Job 654: Subtask second_task
[2021-04-25 00:08:48,895] {logging_mixin.py:104} INFO - Running <TaskInstance: bash_operator_dag.second_task 2021-04-06T00:00:00+00:00 [running]> on host 4a00a306afdc
[2021-04-25 00:08:49,172] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=bash_operator_dag
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-04-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-06T00:00:00+00:00
[2021-04-25 00:08:49,173] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-04-25 00:08:49,176] {bash.py:158} INFO - Running command: echo "testing second bash operator"
[2021-04-25 00:08:49,201] {bash.py:169} INFO - Output:
[2021-04-25 00:08:49,208] {bash.py:173} INFO - testing second bash operator
[2021-04-25 00:08:49,210] {bash.py:177} INFO - Command exited with return code 0
[2021-04-25 00:08:49,389] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=bash_operator_dag, task_id=second_task, execution_date=20210406T000000, start_date=20210425T000848, end_date=20210425T000849
[2021-04-25 00:08:49,563] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-04-25 00:08:49,595] {local_task_job.py:146} INFO - Task exited with return code 0
